pkg/csi/controller.go

```diff
@@ -117,38 +110,47 @@ func (cs *controllerServer) CreateVolume(ctx context.Context,
 	klog.Infof("CreateVolume: requesting volume [%s] with size [%d] MiB, shareable [%v]",
 		diskName, sizeMB, shareable)
 
-	busType := vcdcsiclient.VCDBusTypeSCSI
-	busSubType := vcdcsiclient.VCDBusSubTypeVirtualSCSI
+	var (
+		storageProfile string
+		fsType         string
+		busType        string
+		tuple          vcdcsiclient.BusTuple
+		ok             bool
+	)
+
+	if busType, ok = req.Parameters[BusTypeParameter]; !ok {
+		busType = DefaultBusType
+		klog.Infof("No parameter [%s] specified for raw disk [%s]. Hence defaulting to [%s].", BusTypeParameter, diskName, DefaultBusType)
+	}
+	tuple = vcdcsiclient.BusTypesSet[busType]
```

Comment:
There could be a panic here. I guess we should add a similar check you have added for storageProfile

```
tuple, ok := vcdcsiclient.BusTypesSet[busType]
if !ok {
  return nil, fmt.Errorf("invalid bus type : [%s]", busType)
}
```